user_docs/en/userGuide.t2t

```diff
@@ -59,11 +59,11 @@ This applies to both original and modified copies of this software, plus any der
 For further details, you can [view the full licence. http://www.gnu.org/licenses/old-licenses/gpl-2.0.html]
 
 + System Requirements +
-- Operating Systems: all 32-bit and 64-bit editions of Windows XP, Windows Vista, Windows 7, Windows 8, Windows 8.1 and Windows 10 (including Server operating Systems)
- - For Windows XP 32-bit, NVDA requires Service Pack 2 or higher.
- - For Windows Server 2003, NVDA requires Service Pack 1 or higher.
- - For Windows Vista, NVDA requires Service Pack 2 and the [KB2763674 https://support.microsoft.com/en-us/kb/2763674] update.
- Both of these should be installed if all available updates are applied via Windows Update.
+- Operating Systems: all 32-bit and 64-bit editions of Windows 7, Windows 8, Windows 8.1, Windows 10, and all Server Operating Systems starting from Windows Server 2008 R2.
+
+  - For Windows 7, NVDA requires Service Pack 1 or higher.
+  - For Windows Server 2008 R2, NVDA requires Service Pack 1 or higher.
+ 

```

Comment:
nit: Having a blank line before and after a sub-list is inconsistent with the rest of the User Guide. Also, the other sub-lists are indented with one space rather than two.